extensions/arc/deployment/src/main/java/io/quarkus/arc/deployment/BeanInitializedBuildItem.java

```diff
@@ -0,0 +1,12 @@
+package io.quarkus.arc.deployment;
+
+import io.quarkus.builder.item.MultiBuildItem;
+
+/**
+ * Marker build item to indicate that a build step initializes a CDI bean "manually" through a recorder
+ *
+ * @deprecated use synthetic beans for bean initialization instead
+ */
+@Deprecated
+public final class BeanInitializedBuildItem extends MultiBuildItem {
```

Comment:
I wonder if we should include the "recorder" in the name of the build item. Something like `RecorderBeanInitializedBuildItem`...

Reply:
it sounds a bit confusing. OTOH, the current name doesn't say enough. I'll rename it